test/libsolidity/ASTJSON/assembly/nested_functions.sol

```diff
@@ -0,0 +1,12 @@
+contract C {
```

Comment:
If I'm not mistaken here, this contract is identical to the other in the `syntaxTests` directory, right? Could we remove this file and the JSON output then, because the JSON output is tested for the other file anyways?

Reply:
You mean the JSON export/import test? But that one is a different test that only checks that the results are the same when exporting from an imported AST.. it doesn't check if the AST itself is actually correct

Reply:
Ok, I wasn't sure if that's the case.